poppy/tests/test_fresnel.py

```diff
@@ -446,3 +446,21 @@ def test_fresnel_propagate_direct_2forward_and_back():
     wf.propagate_direct(z)
     wf.propagate_direct(-z)
     np.testing.assert_almost_equal(wf.wavefront, start)
+
+def test_fresnel_return_complex(): 
```

Comment:
consider renaming these two test functions so the names are more parallel, to make it obvious that it's the same basic test for Fraunhofer and Fresnel. `test_return_final_complex_wavefront` and  `test_fresnel_return_final_complex_wavefront`?